simnode/generator.py

```diff
@@ -154,6 +155,51 @@ def add_obstacles_to_scenario(self, scenario: Scenario) -> None:
             elif obstacle_type is Bump:
                 mesh = ProceduralBump(pos, rot, obstacle.width, obstacle.length, height, obstacle.upper_length,
                                       obstacle.upper_width)
+            elif obstacle_type is Stopsign:
```

Comment:
At this point you could reuse the oid instead of generating a random number.

Reply:
oid is None if you don't explicitly define it, so one needs to generate a unique id either in his test generator or DriveBuild takes care of it

Reply:
You're right. I mixed it up with the road ids. In the case of roads I generate names.

Reply:
And I found a bug in the ProceduralBump concerning the name...